File: app/src/main/java/com/androidexamples/app/interfaces/RecyclerViewTouchListener.java
package com.androidexamples.app.interfaces;

import android.view.View;

/**
 * Created by marco on 25/12/2016.
 */

public interface RecyclerViewTouchListener {
    public void onItemClick(View view, int position);
    public void onLongItemClick(View view, int position);
}

File: app/src/main/java/com/androidexamples/app/interfaces/RecyclerViewTouchListener.kt
package com.androidexamples.app.interfaces

import android.view.View

/**
 * Created by marcosvbras on 25/12/2016.
 */

interface RecyclerViewTouchListener {
    fun onItemClick(view: View, position: Int)
    fun onLongItemClick(view: View, position: Int)
}
